shared/actions/wallets.js

```diff
@@ -83,6 +73,23 @@ const buildPayment = (
   })
 }
 
+const spawnBuildPayment = (
+  state: TypedState,
+  action:
+    | WalletsGen.SetBuildingAmountPayload
+    | WalletsGen.SetBuildingCurrencyPayload
+    | WalletsGen.SetBuildingFromPayload
+    | WalletsGen.SetBuildingIsRequestPayload
+    | WalletsGen.SetBuildingToPayload
+    | WalletsGen.DisplayCurrencyReceivedPayload
+) => {
```

Comment:
I funneled these actions into `wallets:buildPayment` so we wouldn't have to keep this list in sync with all the reducer cases that would need to increment the counter